silkrpc/json/types.cpp

```diff
@@ -150,22 +153,44 @@ void to_json(nlohmann::json& json, const Transaction& transaction) {
         json["to"] =  nullptr;
     }
     json["type"] = silkrpc::to_quantity(transaction.type.value_or(0));
+
+    if (transaction.type && transaction.type == silkworm::kEip1559TransactionType) {
+       json["maxPriorityFeePerGas"] = silkrpc::to_quantity(transaction.max_priority_fee_per_gas);
+       json["maxFeePerGas"] = silkrpc::to_quantity(transaction.max_fee_per_gas);
+    }
+    if (transaction.type && transaction.type != 0) {
+       json["chainId"] = silkrpc::to_quantity(*transaction.chain_id);
+       json["v"] = silkrpc::to_quantity((uint64_t)transaction.odd_y_parity);
+       json["accessList"] = transaction.access_list; // EIP2930
+    } else {
+       json["v"] = silkrpc::to_quantity(silkworm::rlp::big_endian(transaction.v()));
+    }
     json["value"] = silkrpc::to_quantity(transaction.value);
-    json["v"] = silkrpc::to_quantity(silkworm::rlp::big_endian(transaction.v()));
     json["r"] = silkrpc::to_quantity(silkworm::rlp::big_endian(transaction.r));
     json["s"] = silkrpc::to_quantity(silkworm::rlp::big_endian(transaction.s));
 }
 
+
+
 } // namespace silkworm
 
 namespace silkrpc {
 
+uint64_t get_block_sizes(const Block &b) {
```

Comment:
I would rename it `get_block_size` and put it in `Block` struct

Reply:
renamed, but Block is struct of silkworm not silkrpc.

Reply:
Don't think so as it's defined in `silkrpc` namespace...

Reply:
Sorry my mistake